pipelines/pipeline_katello_nightly.yml

```diff
@@ -69,6 +69,12 @@
 
 - hosts: pipeline-katello-nightly-centos7
   become: true
+  tasks:
```

Comment:
Why is this needed now? Is it an additional test so we spot issues like the puppetserver one? I'd also like a reboot test but that may be trickier to implement.